deploy/chart/templates/configmap.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mosquitto.fullname" . }}
+  labels: {{- include "mosquitto.labels.standard" . | nindent 4 }}
+    {{- with .Values.mosquitto.common.labels }}
+    {{- toYaml . | trim | nindent 4 }}
+    {{- end }}
+    {{- with .Values.mosquitto.configMap.labels }}
+    {{- toYaml . | trim | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.mosquitto.common.annotations .Values.mosquitto.configMap.annotations }}
+  annotations:
+    {{- with .Values.mosquitto.common.annotations }}
+    {{- toYaml . | trim | nindent 4 }}
+    {{- end }}
+    {{- with .Values.mosquitto.configMap.annotations }}
+    {{- toYaml . | trim | nindent 4 }}
+    {{- end }}
+  {{- end }}
+data:
+  mosquitto.conf: |
+    log_dest stdout
+    {{ .Values.mosquitto.configFile }}
+    {{- if .Values.mosquitto.authentication.passwordFile }}
+    password_file {{ required ".Values.mosquitto.authentication.passwordFilePath is required!" .Values.mosquitto.authentication.passwordFilePath | quote }}
+    {{- end }}

```

Comment:
If there is no password file then you should probably add `allow_anonymous true` since I don't see a way to enable an auth plugin with this chart.

Reply:
@hardillb this is actually possible to set-up by adding necessary values using:

{{ .Values.mosquitto.configFile }}